service-api/src/test/java/greencity/dto/courier/CreateCourierDtoTest.java

```diff
@@ -0,0 +1,57 @@
+package greencity.dto.courier;
+
+import greencity.ModelUtils;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import java.util.Set;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CreateCourierDtoTest {
```

Comment:
please add tests if `nameUk` and `nameEn` contain numbers

Reply:
added